$NetBSD: latex.ltx.diff,v 1.1.1.1 2003/12/24 12:55:03 jmmv Exp $

This is a work around for the '1 year old' error which avoids the format
file from being generated correctly.  Provided by Naoto Shimazaki.

Because of NO_EXTRACT/NO_PATCH/others are defined in teTeX-share{,src}
packages, applying this patch is done by packages' Makefiles on do-install
stage.

--kei

--- latex.ltx.orig	Mon Sep 17 09:24:52 2001
+++ latex.ltx	Mon Sep 17 09:25:16 2001
@@ -72,6 +72,7 @@
   \errmessage
   {LaTeX must be made using an initex with no format preloaded}
 \fi
+\scrollmode
 \catcode`\{=1
 \catcode`\}=2
 \catcode`\#=6
